### ISSUE TITLE
you have been disconnected check your internet connection

### ISSUE THREAD

**Issue by dharmendrakariya** [krakazyabra/jitsi-helm#5]
I am Deploying your helm chart, everything comes up, but when i start meeting it gives, you have been disconnected , check your internet connection.

I searched and found i have to give app_url , so in deployment I defined ENV for App_Url but no help.

any lead on this, have u encountered this ever?

**Comment by krakazyabra** [comment]
Hello. Did you deploy separate STUN/TURN servers? 
What is in logs of JVB, when you face with this error?

**Comment by dharmendrakariya** [comment]
> Hello. Did you deploy separate STUN/TURN servers?

No I dont have TURN server, isnt it inbuilt, I guess, I am not sure actually, I am new to this.

> What is in logs of JVB, when you face with this error?

I cant see JVB

**Comment by dharmendrakariya** [comment]
![image](https://user-images.githubusercontent.com/22043860/118960828-4ab18600-b981-11eb-990f-38a8d93d3fbf.png)

this is ``` shard-0-prosody-``` logs

**Comment by krakazyabra** [comment]
This is prosody, but what about `shard-0-jvb` logs?

**Comment by dharmendrakariya** [comment]
> This is prosody, but what about `shard-0-jvb` logs?

[logs of shard-0-jvb-0 jvb container](https://pastebin.com/raw/GLPgmwuD)

**Comment by dharmendrakariya** [comment]
[logs of shard-1-jvb-0 jvb container](https://pastebin.com/raw/3RgmHMHr)

**Comment by dharmendrakariya** [comment]
btw do we have to setup turn server separately or its actually deployed through this chart only, actually i want to enable video chat in matrix server so i need turn server, and i think jitsi does provide that so. please correct me if i am wrong

**Comment by krakazyabra** [comment]
> do we have to setup turn server separately

Yes, It is strongly recommended. It is not necessary only if you're running jitsi inside local network. The chart doesn't bootstrap turn/stun server, but has values, where you can specify your stun/turn server

**Comment by dharmendrakariya** [comment]
@krakazyabra  okay, I will setup my own turn server, but could you please help me with the above error?

**Comment by MatMaul** [comment]
you should try again with the last version, I strongly suspect this is due to missing `PUBLIC_URL`, and should be fixed now.

**Comment by dharmendrakariya** [comment]
@MatMaul  thanks :) I will try and let you know!!
